### ISSUE TITLE
Accessibility: Missing text alternatives

### ISSUE THREAD

**Issue by sjaeger24** [bycs-lp/mod_learningmap#148]
Several text alternatives are missing ([WCAG 2.1, Guideline 1.1 Text Alternatives](https://www.w3.org/TR/WCAG21/#text-alternatives)):

There is no text alternative describing the image and its interactive purpose. The paths remain unknown for screenreader users. In edit mode, the screenreader provides for places without activites only the URL and the information, that the link is opened in the same window.

**Comment by sh-csg** [comment]
This will be resolved by  #157
